src/hooks/useMellowLPVaults/config.ts

```diff
@@ -24,154 +22,208 @@ const networkConfigurations: { [key: string]: NetworkConfiguration } = {
+          underlyingPools: ['LIDO - ETH'],
+          show: true,
+          soon: false,
           deprecated: true,
         },
       },
     ],
-    MELLOW_ROUTERS: [],
+    MELLOW_ROUTERS: [
+      {
+        router: '0xF875B4BD81b1be40775652d8fDC174512C36DB20',
+        defaultWeights: [100, 0],
+        pivot: 0,
+        metadata: {
+          title: 'MELLOW - USDC',
+          token: 'USDC',
+          maturity: '31 Dec 22',
+          estimatedHistoricApy: '6.51%',
+          description: 'Optimised for LPing across pools in 50x levered positions.',
+          underlyingPools: ['AAVE - USDC LEND'],
+          show: true,
+          soon: false,
+          deprecated: false,
+        },
+      },
+      {
+        router: '0x1963efb3B756e7D17D0e54645339e7E037705cc1',
+        defaultWeights: [100],
+        pivot: 0,
+        metadata: {
+          title: 'MELLOW - ETH',
+          token: 'ETH',
+          maturity: '31 Dec 22',
+          estimatedHistoricApy: '10.96%',
+          description: 'Optimised for Lping across pools in 50x levered positions.',
+          underlyingPools: ['AAVE - ETH BORROW', 'AAVE - ETH LEND', 'LIDO - ETH', 'ROCKET - ETH'],
+          show: true,
+          soon: false,
+          deprecated: false,
+        },
+      },
+      {
+        router: '0xD6e133B9C82F04734B48d5808800078038231a22',
+        defaultWeights: [100],
+        pivot: 0,
+        metadata: {
+          title: 'MELLOW - DAI',
+          token: 'DAI',
+          maturity: '31 Dec 22',
+          estimatedHistoricApy: '10.3%',
+          description: 'Optimised for LPing across pools in 50x levered positions.',
+          underlyingPools: ['AAVE - DAI LEND', 'COMPOUND - DAI LEND'],
+          show: true,
+          soon: false,
+          deprecated: false,
+        },
+      },
+      {
+        router: '0xF875B4BD81b1be40775652d8fDC174512C36DB20',
+        defaultWeights: [0, 100],
+        pivot: 1,
+        metadata: {
+          title: 'MELLOW - USDC',
+          token: 'USDC',
+          maturity: '31 Mar 23',
+          estimatedHistoricApy: '17.66%',
+          description: 'Optimised for LPing across pools in 50x levered positions.',
+          underlyingPools: ['AAVE - USDC BORROW'],
+          show: true,
+          soon: false,
+          deprecated: false,
+        },
+      },
+      {
+        router: '0x0000000000000000000000000000000000000000',
+        defaultWeights: [],
+        pivot: 0,
+        metadata: {
+          title: 'MELLOW - ETH',
+          token: 'ETH',
+          maturity: '31 Mar 23',
+          estimatedHistoricApy: 's00n',
+          description: 'Optimised for LPing across pools in 50x levered positions.',
+          underlyingPools: ['AAVE - ETH BORROW'],
+          show: true,
+          soon: true,
+          deprecated: false,
+        },
+      },
+      {
+        router: '0x0000000000000000000000000000000000000000',
```

Comment:
Is this because Mellow do not support USDT?

Reply:
yes.

Reply:
Thanks. Could you share the testing url in the description as well, please?

Reply:
Done